testing/cucumber/features/linters/scenario_with_bad_name.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Scenario with bad name linter
+
+  As a reader of documentation
+  I want every scenario to have Given and When steps rather than Test, Verify or Check
+  So that the naming is consistent and intent clear
+
+
+  Scenario: Linting
+
+  Note: Also works on outlines.
+
+    Given a linter for tests with bad names
+    And the following feature:
+      """
+      Feature:
+
+        Scenario:
+          This scenario has a bad name
```

Comment:
Copy/past oversight from the TestWithNoNameLinter feature? The example scenario needs to have a bad name, not no name.

Also, because these feature files are used as the documentation for the gem (e.g. [like here](https://app.cucumber.pro/p/52c88528-f03e-448f-8ede-3f9636a22887/cuke_linter/documents/branch/master/testing/cucumber/features/linters/test_with_no_name.feature)), listing several examples of 'bad' words would be a good thing for the users who read it. Using an outline like the  GherkinLint feature did would be fine or, alternatively, you could include an example list in the description of the scenario.

...

In the description of the linting scenario, I mean. Not in the description of the example scenario. Isn't using Cucumber to test Cucumber fun sometimes? ;)